standard/statements.md

```diff
@@ -604,6 +605,19 @@ Local function bodies are always reachable. The endpoint of a local function dec
 
 If the type of the argument to a local function is `dynamic`, the function to be called must be resolved at compile time, not runtime.
 
+A static local function
+
+- May not reference instance members from an implicit or explicit `this` or `base` reference.
+- May reference `static` members from the enclosing scope.
+- May reference `constant` definitions from the enclosing scope.
+- May use `nameof()` to reference locals, parameters, or `this` or `base` from the enclosing scope.
+- May capture state from an enclosing static local function, but may not capture state outside the enclosing static local function. The same rules apply to a lambda.
```

Comment:
```suggestion
- May capture state from an enclosing static local function, but may not capture state outside the enclosing static local function. The same rules apply to a lambda expression.
```

This final sentence feels weird though - are these rules for lambda expressions that are enumerated somewhere else? If so, let's link to them. It not, why not?

Reply:
This is incorrect.  A `static` local function can't capture state.

This should read that a non-static local function can capture state declared in an enclosing static local function:

```csharp
static void M()
{
    OuterM();
    static void OuterM()
    {
        var msg = "hello";
        Console.WriteLine("Hello, World!");

        InnerM();
        void InnerM() // not static
        {
            Console.WriteLine(msg);
        }
    }
}
```

Reply:
Great - @BillWagner are you happy to propose a change for this?